README.md

```diff
@@ -129,4 +129,19 @@ Bump versions and publish with:
 npm publish --access public
 ```
 
+## Ignore Files
+
+Create a file called `.claspignore` in the root directory of your Apps Script project. Add files as if it were a .gitignore, and they will be excluded from `clasp push`, `clasp deploy`, etc...  
+
+As of now, clasp does not support pushing only tracked files, so to push a few files from an npm project with bundling, for example, a sample `.claspignore` could look like:
+
+```
+**/**
+!build/Main.gs
+!appsscript.json
+```  
+This file ignores everything but the manifest and the bundle.
+
+Note: the `.claspignore` file is parsed with [Anymatch](https://github.com/micromatch/anymatch), making it match files differently from a typical `.gitignore`, especially with directories. To ignore a directory, use syntax like: `**/node_modules/**`
```

Comment:
> "use syntax like: `**/node_modules/**`"

You already have a colon for "Note:". Just say:

use syntax like `**/node_modules/**`.